doc/changes.rst

```diff
@@ -1,6 +1,82 @@
 Changelog
 =========
 
+Release v0.2.0 (2020/09/11)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.3781524
```

Comment:
Just to make sure someone (other than me) can do it, can one of @GenericMappingTools/python reserve a new version DOI under the PyGMT banner at https://zenodo.org/record/3930577? Just click on the green "New version" button (see sceenshot).

![PyGMT Zenodo page](https://user-images.githubusercontent.com/23487320/92861143-3958c700-f44d-11ea-979a-30d599713ec0.png)

This will take you to a page where there should be a 'Reserve DOI' button somewhere. Click on that, and copy the DOI link here!

Reply:
![image](https://user-images.githubusercontent.com/3974108/92864133-c3ac1700-f3ca-11ea-83fe-c0fc68c6860e.png)

I can't see it.

Reply:
Hmm, did you login with Github? I'm not quite sure how the permissions work on Zenodo. Might need to email them.

Reply:
Yes, I log in using my github acccout. I can see the PyGMT project in my account, but it's off by default.
![image](https://user-images.githubusercontent.com/3974108/92973621-9145fc80-f452-11ea-81fb-ba94993be41c.png).

After turning it on, I see:
![image](https://user-images.githubusercontent.com/3974108/92973708-bfc3d780-f452-11ea-9d19-89ac76891154.png)

Reply:
Ok, bit of a long story, but there were some double ups with the PyGMT release DOIs because I tried to reserve a DOI before a release, and put that in the release notes, **but** when we make a release, the automated Github integration would just ignore that reserved DOI and make a new one. So I had to email Zenodo and delete those duplicated DOIs. (Will forward you the email chain in a bit).

Reply:
Ok email sent. Just to clarify, there's 2 options that I know of to make a Zenodo DOI, each with pros/cons (see also FAQs at https://help.zenodo.org/):

1. Use the **automated** Github integration:
  - Pro: It's automated, publishing a release creates the DOI automatiically
  - Con: We don't know the DOI number beforehand, so can't put it into the release notes. Also, I couldn't manage to link it to our current permanent/concept DOI at https://doi.org/10.5281/zenodo.3781524.

2. **Manually** reserve a DOI (what was done for PyGMT v0.1.x):
  - Pro: We know the DOI beforehand, so can put it into our release notes. Also more control with the metadata information.
  - Con: It's not automated, I had to manually download the pygmt-v0.1.x.zip file, and upload it to Zenodo. Also as we found out, maybe it's linked only to my account?!!

So what should we do here?

Reply:
Maybe I'll just reserve the DOI again for this v0.2.0 release, and you (@seisman) or someone else can do it next time, since it might take a few days to email Zenodo and sort things out. Don't want to delay this release any longer.

Reply:
OK to me.